data/json/items/comestibles.json

```diff
@@ -1725,8 +1725,8 @@
     "comestible_type": "FOOD",
     "symbol": "%",
     "quench": -10,
-    "healthy": -3,
-    "calories": 1943,
+    "healthy": -5,
+    "calories": 1923,
```

Comment:
Do we really care about 20 kCal difference? that's about 2 grams of fat difference, or 1%.

Reply:
It was a lot more before I got better information. As it stands yeah this change is pretty small and the original is pretty close to the correct value. I can remove the change in calories but I would think we want it to be the correct value?

Reply:
I'm not sure that either value is "correct" because I suspect the error bars on the calorie density of "animal fat" are larger than 1%.  No need to revise your PR; this is fine.